src/components/Slider/README.md

```diff
@@ -190,143 +190,96 @@ LANDING_BLOCK-->
+<Slider marksCount={11} />
 ```
 
 <!--/GITHUB_BLOCK-->
 
-`0` or empty array `[]` value in `marks` property hide all marks from `Slider`.
+### Available values
 
-<!--LANDING_BLOCK
-<ExampleBlock
-    code={`
-<Slider marks={0} />
-`}
->
-    <UIKitExamples.SliderExample marks={0} />
-</ExampleBlock>
-LANDING_BLOCK-->
-
-<!--GITHUB_BLOCK-->
-
-```tsx
-<Slider marks={0} />
-```
-
-<!--/GITHUB_BLOCK-->
+The `availableValues` property is used to define specific values the slider can handle, as opposed to a continuous range. This is particularly useful when only certain discrete values are valid for selection. `availableValues` enables specifying an array of numbers which are the exact values that the users are allowed to select when working with the `Slider`.
 
-> The mark value is available for selection, even if it does not match the `step` value condition
-
-You are able to change display format of marks values by using `marksFormat` property.
-
-#### Define available values
-
-You can set `step` property to `null` to define a set of specific values that the slider can handle, as opposed to a continuous range. This is particularly useful when only certain discrete values are valid for selection. In that case properties `min`, `max` and `marks` allows specifying an array of numbers representing the exact values that users are allowed to select using the `Slider`.
+> The `availableValues` property overrides `min`, `max`, `marksCount` and `step` (when those are used all at the same time); in this case, the slider will directly use the provided array values rather than a continuous range.
 
 <!--LANDING_BLOCK
 <ExampleBlock
     code={`
-<Slider marks={[10, 20, 50, 55, 65, 80]} step={null}/>
+<Slider availableValues={[10, 20, 50, 55, 65, 80]} />
 `}
 >
-    <UIKitExamples.SliderExample marks={[10, 20, 50, 55, 65, 80]} step={null}/>
+    <UIKitExamples.SliderExample availableValues={[10, 20, 50, 55, 65, 80]} />
 </ExampleBlock>
 LANDING_BLOCK-->
 
 <!--GITHUB_BLOCK-->
 
 ```tsx
-<Slider marks={[10, 20, 50, 55, 65, 80]} step={null} />
+<Slider availableValues={[10, 20, 50, 55, 65, 80]} />
 ```
 
 <!--/GITHUB_BLOCK-->
 
 ## Tooltip
 
-The `tooltipDisplay` property is used in `Slider` component to control the display behaviour of a tooltip that shows the current value as the user interacts with the slider. `auto` value shows tooltip only when `Slider`'s handle are hovered by cursor or focused.
+The `hasTooltip` property is a boolean attribute. It toggles displaying the tooltip that shows the current value as the user is moving with the slider.
 
 <!--LANDING_BLOCK
 <ExampleBlock
     code={`
-<Slider tooltipDisplay="on" />
+<Slider hasTooltip={true} />
 `}
 >
-    <UIKitExamples.SliderExample tooltipDisplay="on" />
+    <UIKitExamples.SliderExample hasTooltip={true} />
 </ExampleBlock>
 LANDING_BLOCK-->
 
 <!--GITHUB_BLOCK-->
 
 ```tsx
-<Slider tooltipDisplay="on" />
+<Slider hasTooltip={true} />
 ```
 
 <!--/GITHUB_BLOCK-->
 
```

Comment:
Check Slider's description one more time please. You translated an old version. 
Now `hasTooltip`, `availableValues` and `marksCount` are deprecated. 
So the table with properties is also has been changed.

Reply:
Hi, fixed, please recheck!